advanced/Scripts/piholeLogFlush.sh

```diff
@@ -50,87 +81,16 @@ if [[ "$*" == *"once"* ]]; then
         fi
         /usr/sbin/logrotate --force --state "${STATEFILE}" /etc/pihole/logrotate
     else
-        # Copy pihole.log over to pihole.log.1
```

Comment:
Can we preserve the comment please (or move it into the new function)? I think the info why we don't just move to a new location is valuable to be preserved for future readers.